projects/dotcms-field-elements/src/index.html

```diff
@@ -185,16 +185,32 @@
                 required="true"
             ></dot-binary-file>
 
+            <div>
```

Comment:
Do we need this `<div>`?

Reply:
just using it to avoid the rule `.wrapper > *` in the index.html